test/e2e/scheduling/priorities.go

```diff
@@ -163,7 +165,9 @@ var _ = framework.KubeDescribe("SchedulerPriorities [Serial]", func() {
 		defer framework.RemoveLabelOffNode(cs, nodeName, k)
 
 		// make the nodes have balanced cpu,mem usage ratio
-		createBalancedPodForNodes(f, cs, ns, nodeList.Items, podRequestedResource, 0.6)
+		err := createBalancedPodForNodes(f, cs, ns, nodeList.Items, podRequestedResource, 0.6)
+		framework.ExpectNoError(err)
+
```

Comment:
I found that most code do not include a blank line for such case, ditto for others